### PR TITLE
Update chopsticks for v0.6.4

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

Original PR: https://github.com/moonbeam-foundation/moonbeam-docs/pull/668

Need to translate english lines

### Checklist

- [ ] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [ ] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [ ] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

File: builders/build/substrate-api/chopsticks.md [hunk 1]
```diff
@@ -13,113 +13,194 @@ description: 学习如何通过Chopsticks重放区块、解析状态更迭、测
...
+    npx @acala-network/chopsticks \
+    --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonbeam.yml
     ```
 
-=== "Moonbase Alpha"
+=== "Local File Path"
+
+    ```bash
+    npx @acala-network/chopsticks --config=configs/moonbeam.yml
     ```
```
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:153]: Still waiting on translation of these tab headers

File: builders/build/substrate-api/chopsticks.md [hunk 2]
```diff
@@ -134,58 +215,83 @@ import-storage:
...
+|       方法       |         参数          |                    描述                    |
+|:----------------:|:---------------------:|:------------------------------------------:|
+|  `dev_newBlock`  |       `options`       |           生成一个或多个新区块。           |
+| `dev_setStorage` | `values`, `blockHash` |          创建或覆盖任何存储的值。          |
+| `dev_timeTravel` |        `date`         |       将区块的时间戳设置为`date`值。       |
+|  `dev_setHead`   |    `hashOrNumber`     | 将区块链的块头设置为特定的哈希值或区块号。 |
+
+The parameters above are formatted in the following ways:  
+
+|   Parameter    |               Format                |                                Example                                 |
```
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:278]: Still waiting on translation of these table headers

File: builders/build/substrate-api/chopsticks.md [hunk 3]
```diff
@@ -13,113 +13,194 @@ description: 学习如何通过Chopsticks重放区块、解析状态更迭、测
...
+|      `import-storage`      |             预先定义的JSON/YAML存储文件路径以覆盖平行链存储。              |
+| `allow-unresolved-imports` |              在使用WASM构建平行链时是否允许WASM未解析的导入。              |
+|           `html`           |                  包含此标志以生成区块之间存储差异的预览。                  |
+|   `mock-signature-host`    |  模拟签名主机，让任何以0xdeadbeef开头并由0xcd填充的签名都被认为是有效的。  |
+
+您可以将配置文件与基本指令`npx @acala-network/chopsticks`结合使用，通过为其提供`--config`标志来分叉资产。
+
+您可以使用默认配置文件的原始GitHub URL、本地配置文件的路径，或者仅使用链的名称作为`--config`标志。例如，以下指令都以相同的方式使用Moonbeam的配置：
+
+=== "Chain Name"
```
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:136]:
```suggestion
=== "链名称"
```

File: builders/build/substrate-api/chopsticks.md [hunk 4]
```diff
@@ -13,113 +13,194 @@ description: 学习如何通过Chopsticks重放区块、解析状态更迭、测
...
+
+=== "GitHub URL"
+
+    ```bash
+    npx @acala-network/chopsticks \
+    --config=https://raw.githubusercontent.com/AcalaNetwork/chopsticks/master/configs/moonbeam.yml
     ```
 
-=== "Moonbase Alpha"
+=== "Local File Path"
```
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:149]:
```suggestion
=== "本地文件路径"
```

File: builders/build/substrate-api/chopsticks.md (hunk 2 above)
Comment (eshaben) [inline builders/build/substrate-api/chopsticks.md:278]:
```suggestion
|   参数    |               格式                |                                示例                                 |
```

**Review by themacexpert — APPROVED** [review]
LGTM 👍

**Pull request merged**